doc/user_manual/generated/sklRom.tex

```diff
@@ -5402,7 +5402,7 @@ \subsubsection{SVC}
 
     \item \xmlNode{class\_weight}: \xmlDesc{[balanced]}, 
       If not given, all classes are supposed to have weight one.
-      The “balanced” mode uses the values of y to automatically adjust weights
+      The �balanced� mode uses the values of y to automatically adjust weights
```

Comment:
All the changes in this file is not show correctly. I suggest you copy this file from RAVEN devel, and replace this one instead to generate it directly using the script. It seems to me there is a problem in the script when generate this file. @wanghy-anl

Reply:
Thanks for the suggestion. I copied the file from devel branch. Please check the latest push 070a9989ebc1d514d4c85f51ae9150de4d1eca44. @wangcj05